packages/grid/x-grid-modules/src/models/gridOptions.tsx

```diff
@@ -66,11 +66,11 @@ export interface GridOptions {
    */
   columnBuffer: number;
   /**
-   * Enable multiple selection using the CTRL or CMD key.
+   * Disable multiple selection using the CTRL or CMD key.
```

Comment:
```suggestion
   * If `true`, multiple selection using the CTRL or CMD key is disabled.
```

Use standard props documentation wording for boolean props. (The "If `true`" part at least - there doesn't seem to be consistency as to whether the outcome should be present or future tense ("is" / "will be")

Reply:
> present or future tense ("is" / "will be")

I have noticed this inconsistency too yesterday when reviewing the date picker pull request migration to the main repository. Should we settle on the present?